incubator/uchiwa/README.md

```diff
@@ -0,0 +1,86 @@
+# Uchiwa
+
+[Uchiwa](https://uchiwa.io) is a simple yet effective open-source dashboard for the Sensu monitoring framework.
+
+
+## TL;DR;
+
+```console
+$ helm install incubator/uchiwa
```

Comment:
Since you aren't using any alpha features and have NOTES, you can put this straight into stable.